.github/workflows/main.yml

```diff
@@ -25,40 +25,56 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    # needed java 1.11 for checkstyle
```

Comment:
```suggestion
    # Java 11 is specifically needed for Checkstyle.
```

Reworded, and changed casing for consistency.

Reply:
done

Reply:
resolving on behalf of ben as this is a straight change.